src/code/templates/egg-sort-game.js

```diff
@@ -29,6 +29,13 @@ export default class EggSortGame extends Component {
     onEggPlaced: PropTypes.func.isRequired
   }
 
+  state = {
+    selectedBaskets: [],
+    clickedBasket: null,
+    selectedEggIndex: null,
+    selectedEgg: null
+  }
+
```

Comment:
I believe it's [the ES6+ way](https://babeljs.io/blog/2015/06/07/react-on-es6-plus#property-initializers).